### PR TITLE
Allow disabling of NFS services

### PR DESCRIPTION
**Pull request by pprkut**
### PR progress checklist (to be filled in by reviewers)

- [x] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [x] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [x] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

Allow the formula to disable NFS on servers that shouldn't have it.

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [x] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: pillar.example [hunk 1]
```diff
@@ -5,6 +5,7 @@ nfs:
   mount_opts: noauto,ro
   persist_unmount: true
   persist_mount: false
+  disabled: false
```
Comment (baby-gnu) [inline pillar.example]: I think I prefer to have `enabled: false` like in many other formulas.

**Review by baby-gnu — CHANGES_REQUESTED** [review]

File: pillar.example (hunk 1 above)
Comment (pprkut) [reply pillar.example]: I suppose `enabled: true` then, so NFS remains enabled by default unless disabled in configuration?

Comment (baby-gnu) [reply pillar.example]: Yes, the key `enabled` is better. It avoid the reverse logic `the service is enabled when disabled is false`.

**Review by baby-gnu — APPROVED** [review]
Thanks for the update.

**Review by myii — APPROVED** [review]

**Comment by myii** [comment]
Merged, thanks for the PR, @pprkut.
Appreciate the review, @baby-gnu.

**Pull request merged**